apps/cli/src/node/docker-compose-anvil.yaml

```diff
@@ -19,27 +19,29 @@ services:
             - 8545:8545
 
     dapp_deployer:
-        image: cartesi/sdk:0.12.0-alpha.1
+        image: cartesi/rollups-node:devel
         restart: on-failure
         depends_on:
-            anvil:
+            validator:
                 condition: service_started
+        env_file:
+            - ${CARTESI_BIN_PATH}/node/default.env
         command:
-            [
-                "cast",
-                "send",
-                "--rpc-url",
-                "http://anvil:8545",
-                "--private-key",
-                "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
-                "0x4C11C7F82D6D56a726f9B53dd99af031AFd86BB6",
-                "deployContracts(address,address,bytes32,bytes32)",
-                "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
-                "720",
-                "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
-                "0x0000000000000000000000000000000000000000000000000000000000000000",
-                "0x0000000000000000000000000000000000000000000000000000000000000000",
-            ]
+            - /bin/bash
```

Comment:
I usually don't like to use `bash` as the command, and instead use the actual process we need to start.
Is this because of env variables?
I understand as well that we need the command stdout to go to a file, in the host `.cartesi` directory. How are we doing that?

Reply:
Yeah, it was a way to debug with -x

About saving do the host, we'd need to mount it into the container and redirect to a file.

Or we'd need to save to a named volume and copy on another process with docker cp.

Reply:
mounting the volume is easy, but how to redirect to a file? Does the following work??
```
- ">"
- "filename"
```

Reply:
Support for environment variables will be added to the cartesi-rollups-cli